frontend/src/components/Quiz/Result.jsx

```
@@ -0,0 +1,136 @@
+                  document.getElementById(`question-${index}`).classList.toggle('hidden');
+                };
+
+                return (
+                  <div 
+                    key={index} 
+                    className={`border rounded-lg overflow-hidden ${
+                      q.is_correct ? 'border-green-500' : 'border-red-500'
+                    }`}
+                  >
+                    <div 
+                      role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)